www/nginx/src/opnsense/scripts/nginx/setup.php

```diff
@@ -75,6 +75,7 @@ function find_ca($refid)
 @chgrp('/var/db/nginx/auth', GROUP_OWNER);
 @chgrp('/var/log/nginx', GROUP_OWNER);
 $nginx = $config['OPNsense']['Nginx'];
+openlog("configd.py", LOG_ODELAY, LOG_USER);
```

Comment:
@AdSchellevis I guess we should not use the prefix of configd here. The script is run on init (rc.d) and by configd action. Maybe use `nginx` as prefix

Reply:
@fabianfrz I am trying to log to the backend log here. `configd.py` is the filter on syslog-ng settings for the messages destined for this log

Reply:
@fabianfrz @AdSchellevis 
Hi! what if i try to use syslog for Global Error Log at
https://github.com/opnsense/plugins/blob/cc42b5287871edb4dabd8384841ef3ad44705d4f/www/nginx/src/opnsense/service/templates/OPNsense/Nginx/nginx.conf#L15
repace with (`error_log  syslog:server=unix:/var/run/log;`)
and send log messages to it from setup.php too (replace tag to 'nginx').
would that be ok?

Reply:
@fabianfrz @AdSchellevis 
Hi! what if i try to use syslog for Global Error Log at
https://github.com/opnsense/plugins/blob/cc42b5287871edb4dabd8384841ef3ad44705d4f/www/nginx/src/opnsense/service/templates/OPNsense/Nginx/nginx.conf#L15
repace with (`error_log  syslog:server=unix:/var/run/log;`)
and send log messages to it from setup.php too (replace tag to 'nginx').
would that be ok?

Reply:
@kulikov-a my preference would always be to use syslog, but I'm not involved in the nginx plugin

Reply:
@AdSchellevis Thanks! Your opinion matters anyway ) I'll try and see if it works correctly

Reply:
@AdSchellevis @fabianfrz how about https://github.com/opnsense/plugins/pull/3205/commits/844b859609a6b9d83c33a05525587491ddfef37e then?
Unfortunately, simply using the configuration option to use syslog has the same disadvantages as in the unbound case: syslog logging only starts to work after a successful start. and thus we lose debug startup error messages (https://github.com/opnsense/core/pull/5243). so I had to parse the file.
will that go?

Reply:
@kulikov-a  I don't really like the error log magic, but I'm also not sure why you need it to be honest. Can't nginx send the output to syslog, or only in flagged as debug?

Reply:
@AdSchellevis maybe I didn't fully explain. nginx **can** send messages to the syslog by itself. but it starts to do this only after the master process has **successfully** loaded (that is, there should be no errors in the config file, the sockets is free etc.). if an error occurs at the start, then the diagnostic message (like "Invalid variable name ..") does not get into the log (hard to understand the reason for the failure). in the case of file logging, all startup errors gets into the log.
(this behavior resembles unbound).

Without file-to-syslog-magic, separate diagnostic scripts will be required (at least config file testing imho).
or may be we can just leave the initial option (keep logging to file and send setup.php diagnostic messages to the backend log ("configd.py" prefix))

Reply:
@kulikov-a I understand your problem, but in case of unbound it's just a feed to `logger` which is quite clean, in this case it has hard to follow magic, it might be cleaner to feed information back in the startup/apply via the gui as it's most likely an issue was caused there. 

But as I mentioned earlier, I'm not involved in the Nginx plugin (nor do I want to be), any solution likely works for me.

Reply:
@AdSchellevis sorry for bother. I just decided that if it affects the core componentsl, then it is better to get your opinion. if you're not happy with this syslog "magic", then I'll look into other options. and if you and @fabianfrz  don't mind just using the `configd.py` tag in setup script, then I would stop there for now (I'll try to add at least a "config test" to the gui before migrating to syslog).
testing small changes in the processhandler, which should allow to get the stderr of configd commands in the backend log (https://github.com/kulikov-a/core/commit/884c8bedf91b7ff0bf4ed7e6e3e51d02f91e0c84) and . seems to work

Reply:
@kulikov-a help? I think @fabianfrz suggestion about the target was better, technically this isn't a `configd` message. I kind of got distracted by the rest of the fuzz in the ticket. I guess at some point in time I do need to spend some time on `configd` itself as well, still have some wishes on my list, but priority hasn't been high enough yet.  As usual, if additions are small and easy to verify (and add something useful), we might merge it, sometimes it just takes time.

Reply:
@AdSchellevis hi! yes, thanks! sorry again - i distracted you and I myself seem to be a little confused (although tbh i was glad when I came up with the parser ) ). I will update the pr tomorrow with the standard use of syslog and add a config check with syslog feeding in rc.d conf. this will at least add information about config errors to the log (other startup errors will hopefully end up in the backend log).
Thanks!